Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.6.9-alpine3.10
+LABEL maintainer="Hypothes.is Project and contributors"
+
+# Install nginx & supervisor
+RUN apk add --no-cache nginx gettext supervisor \
+    && mv /usr/bin/envsubst /usr/local/bin/envsubst \
```

Comment:
?

Reply:
It was having trouble finding the command without this and I pulled this from the official nginx image. I didn't bother thinking to deeply about it at the time as I was just trying to get something working first.

Reply:
I've removed this and specified an absolute path.